src/App.css

```diff
@@ -1,8 +1,9 @@
 #quote {
   width: 26rem;
-  margin: 3rem 0;
+  margin: 3rem 20rem;
   text-align: center;
   padding: 1rem;
+  font-size: 1.5rem;
 }
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍